apps/frontend/src/components/global/Snackbar.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <v-snackbar v-if="show" id="info-snackbar" v-model="show" timeout="10000">
+    <template v-if="error">
+      <template v-if="message" id="snackbar-message"
+        >ERROR: {{ message }}</template
```

Comment:
The old Toasted display was modified to not include `ERROR:` in front of everything. It might be best to remove the `ERROR:` from the message below as well.